commerce_coordinator/apps/core/constants.py

```diff
@@ -17,3 +17,7 @@ class PaymentState(Enum):
     FAILED = 'failed'
     PENDING = 'pending'
     PROCESSING = 'processing'
+
+
+class PaymentMethod(Enum):
+    STRIPE = 'stripe'
```

Comment:
@RylanMorrison 

In the API spec [here](https://github.com/edx/commerce-coordinator-private/blob/main/docs/api-spec/theseus-api-spec.md#enums-1), There is only one enum value for `PaymentMethodName` i.e `Stripe No Capture`. Is there any specific reason to name it `Stripe No Capture`? Do we support any other value for this Enum?

Reply:
We originally went with `Stripe No Capture` to differentiate it from the Stripe payment method we use on the frontend and because we don't want Spree to attempt to process the payment Titan receives (ie send it to Stripe). It turns out flagging the payment method as `no capture` isn't enough to prevent Spree from attempting to process it and I'm currently looking at resolving this in [THES-159](https://2u-internal.atlassian.net/browse/THES-159)

This means this value will change, likely to something like `edX Stripe`, and that is the only value we would support. Maybe there would be other values when we get to supporting other payment providers